src/Raven.Client/Documents/Queries/Timings/QueryTimings.cs

```diff
@@ -5,10 +5,19 @@
 
 namespace Raven.Client.Documents.Queries.Timings
 {
+    /// <summary>
+    ///     Representation of query timings.
+    /// </summary>
     public sealed class QueryTimings : IFillFromBlittableJson, IDynamicJson
     {
+        /// <summary>
+        ///     Total time spent on query execution in milliseconds.
```

Comment:
This is server side execution, right? Maybe we should say that explicitly?

Reply:
Yes, I added this to documentation